src/OrleansAzureUtils/Providers/Storage/AzureBlobStorage.cs

```diff
@@ -64,7 +67,9 @@ public class AzureBlobStorage : IStorageProvider
         /// <see cref="IProvider.Init"/>
         public async Task Init(string name, IProviderRuntime providerRuntime, IProviderConfiguration config)
         {
-            Log = providerRuntime.GetLogger("Storage.AzureBlobStorage");
+            var loggerFactory = providerRuntime.ServiceProvider.GetRequiredService<ILoggerFactory>();
+            this.logger = loggerFactory.CreateLogger<AzureBlobStorage>();
+            Log = new LoggerWrapper(this.GetType().FullName, loggerFactory);
```

Comment:
not sure we should preserve this property in the interface, as we'll never be able to move it to Legacy.
If we do preserve it (even if temporarily), just add another constructor to LoggerWrapper that takes an ILogger, because in this particular case you already created an ILogger for it, no need to create a new one.

Reply:
I think the Logger in the interface also serves as a LoggerFactory, because of its GetLogger method. LoggerWrapper takes a loggerFactory for that. This PR is "remove internal usage of Logger in AzureUtils", which is meant to be a brain dead logging migration. Whether we should preserve the Logger property in the interface is not in the scope of this PR. we can discuss it in another PR, something like "remove/obsolete Logger in IStoerageProvider" interface.

Reply:
Ok, sure, but you can still have an overload that takes an ILogger AND ILoggerFactory. What I was trying to avoid there is to create yet a new instead of ILogger inside the wrapper, when it's not needed.